group/lc/main.tf

```diff
@@ -0,0 +1,74 @@
+# AWS Launch Configuration
+
+## Creates security group
+resource "aws_security_group" "sg_asg" {
+  name_prefix = "${var.stack_item_label}-asg-"
+  description = "${var.stack_item_fullname} security group"
+  vpc_id      = "${var.vpc_id}"
+
+  tags {
+    Name        = "${var.stack_item_label}-asg"
+    application = "${var.stack_item_fullname}"
+    managed_by  = "terraform"
+  }
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+## Creates launch configuration
+resource "aws_launch_configuration" "lc" {
+  count                       = "${index(split(",",var.opposite),signum(length(var.ebs_snapshot_id)))}"
```

Comment:
😸  It's cute, but i think `"${signum(length(var.ebs_snapshot_id)) + 1 % 2}"` might read more legibly